docs/src/AlgebraicGeometry/ToricVarieties/BlowdownMorphisms.md

```diff
@@ -2,50 +2,41 @@
+This will also construct the underlying toric morphism. We can specify the name for the coordinate in the Cox ring that is assigned to `new_ray` using the optional argument `coordinate_name`.
 
-## Constructors
-
-Blowups of toric varieties are obtained from star subdivisions of polyhedral fans. In the most general form,
-a star subdivision is defined by a new primitive element in the fan. Below, we refer to this new primitive
-element as `new_ray`. In addition to this `new_ray`, our design of toric blowdown morphisms requires an
-underlying toric morphism. With an eye towards covered schemes as possible return value, any toric blowdown
-morphism must also know (to compute) its blowup center in the form of an ideal sheaf. The following constructor
-allows to set this ideal sheaf center upon construction:
-- `toric_blowdown_morphism(bl::ToricMorphism, new_ray::AbstractVector{<:IntegerUnion}, center::AbsIdealSheaf)`
-The "working-horse" constructor however is the following:
-- `toric_blowdown_morphism(Y::NormalToricVariety, new_ray::AbstractVector{<:IntegerUnion}, coordinate_name::String)`
-This constructor will, among others, construct the underlying toric morphism. In addition, we can then specify
-a name for the coordinate in the Cox ring that is assigned to `new_ray`.
+More generally, we can construct a blowup along a closed subscheme given by an ideal in the Cox ring or by an ideal sheaf of the corresponding covered scheme. In general, this will result in a non-toric variety.
 
 
+## Constructors
 
-## Blowdown morphisms from blowing up toric varieties
-
-The following methods blow up toric varieties. The center of the blowup can be provided in different formats.
+The following methods blow up toric varieties. The closed subscheme along which the blowup is constructed can be provided in different formats.
 We discuss the methods in ascending generality.
 
-For our most specialized blow-up method, we focus on the n-th cone in the fan of the variety `v` in question.
-This cone need not be maximal! The ensuing star subdivision will subdivide this cone about its "diagonal"
+For our most specialized blowup method, we focus on the n-th cone in the fan of the variety `v` in question.
+This cone need not be maximal. The ensuing star subdivision will subdivide this cone about its "diagonal"
 (the sum of all its rays). The result of this will always be a toric variety:
 ```@docs
 blow_up(v::NormalToricVariety, n::Int; coordinate_name::String = "e")
 ```
-More generally, we can provide a primitive element in the fan of the variety in question. The resulting
-star subdivision leads to a polyhedral fan, or put differently, the blow-up along this center is always toric:
+More generally, we can provide a primitive element in the fan of the variety in question and construct a toric morphism as in Section 11.1 Star Subdivisions in [CLS11](@cite). The resulting
+star subdivision leads to a polyhedral fan, or put differently, the blowup is always toric:
```

Comment:
Here is another such line break. I will stop here. Please take a look (if you care to fix this).

Reply:
Fixed.